### PR TITLE
S17

### PR DESCRIPTION
**Pull request by devwistkey**
@tralkan

### CODE REVIEW

File: components/VerifiableCredentials/Pause/index.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import styles from './styles.module.scss'
```
Comment (tralkan) [inline components/VerifiableCredentials/Pause/index.tsx:1]: how can we have the Pause & Unpause components implemented globally so they work for all xWallets? @devwistkey

File: components/ZilPay/zilpay-base.ts [hunk 2]
```diff
@@ -5,336 +5,336 @@ import { toast } from 'react-toastify'
 import { updateShowZilpay } from '../../src/store/modal'
 
 type Params = {
-  contractAddress: string
-  transition: string
```
Comment (tralkan) [inline components/ZilPay/zilpay-base.ts:9]: do we have a new config for prettier? this file shouldn't have changed @devwistkey @sherly09

File: pages/[username]/vc/index.tsx [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
                     <Headline data={data} />
                 </div>
-                <XPoints />
+                <VerifiableCredentials />
             </Layout>
         </>
     )
 }
 
+export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
```
Comment (tralkan) [inline pages/[username]/vc/index.tsx:26]: why do we have 2 getStaticPaths? @devwistkey

**Review by tralkan — APPROVED** [review]

File: pages/[username]/vc/index.tsx (hunk 3 above)
Comment (devwistkey) [reply pages/[username]/vc/index.tsx:26]: Hi @tralkan we only have 1 getStaticPaths, the second one ":GetStaticPaths" is just only typescript declaration. Smae like :string :number etc

File: components/ZilPay/zilpay-base.ts (hunk 2 above)
Comment (devwistkey) [reply components/ZilPay/zilpay-base.ts:9]: Hi @tralkan, I don't have any changes for prettier. What version are you using? Mine v2.6.2

File: components/VerifiableCredentials/Pause/index.tsx (hunk 1 above)
Comment (devwistkey) [reply components/VerifiableCredentials/Pause/index.tsx:1]: Sure, let me add this on notion @tralkan

File: components/ZilPay/zilpay-base.ts (hunk 2 above)
Comment (tralkan) [reply components/ZilPay/zilpay-base.ts:9]: What about the indentation? To set the same for everyone

Comment (devwistkey) [reply components/ZilPay/zilpay-base.ts:9]: But the others have the same indentation. Ex https://i.postimg.cc/HsyWyvNY/Screen-Shot-2022-09-16-at-09-56-09.png

**Pull request merged**